packages/active-campaign-client/src/handlers/addContact.ts

```diff
@@ -0,0 +1,47 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import { acClient } from '../activeCampaignClient';
+import { SignUpUserData } from 'nextjs-website/src/lib/types/sign-up';
+import { ACContactPayload } from '../activeCampaign';
+
+export type HandlerEvent = Pick<APIGatewayProxyEvent, 'headers' | 'body'>;
```

Comment:
```suggestion
export type HandlerEvent = Pick<APIGatewayProxyEvent, 'body'>;
```
Like suggest in  packages/active-campaign-client/src/__tests__/handlers/addContact.test.ts
We still need the headers ? Because Cognito should be checked by API gateway and here `headers` are not used or i am missing something ?